### PR TITLE
fix(daemon): makeIdentifiedHost wrapped controller bug

### PR DESCRIPTION
**Pull request by kriskowal**
closes: #1901

## Description

A flaw in the Endo pet daemon prevented the user from creating new user profiles. The flow crept in through the refactor that separated external and internal facets for every entity that endo can create.

### CODE REVIEW

File: packages/daemon/test/test-endo.js [hunk 1]
```diff
@@ -670,3 +670,27 @@ test('terminate because of requested capability', async t => {
...
+    'client',
+    locator.sockPath,
+    cancelled,
+  );
+  const bootstrap = getBootstrap();
+  const host = E(bootstrap).host();
+  const host2 = E(host).provideHost('fellow-host');
+  await E(host2).makeWorker('w1');
+  const ten = await E(host2).evaluate('w1', '10', [], []);
+  t.is(10, ten);
```
Comment (dckc) [inline packages/daemon/test/test-endo.js:693]: `t.is` calls its args actual, expected, so this looks backward. Not that it matters much.

```suggestion
  t.is(ten, 10);
```

File: packages/daemon/src/daemon.js [hunk 2]
```diff
@@ -445,13 +445,12 @@ const makeEndoBootstrap = (
       const workerFormulaIdentifier = `worker-id512:${formulaNumber}`;
       // Behold, recursion:
       // eslint-disable-next-line no-use-before-define
-      const external = makeIdentifiedHost(
+      return makeIdentifiedHost(
```
Comment (dckc) [inline packages/daemon/src/daemon.js:448]: How did this pass static type-checking previously? I guess type coverage isn't thorough?

**Review by dckc — APPROVED** [review]
good stuff

File: packages/daemon/src/daemon.js (hunk 2 above)
Comment (kriskowal) [reply packages/daemon/src/daemon.js:448]: I’m betting we have to disable implicit any to get more juice from ts here.

**Pull request merged**